packages/pangea-sdk/src/test/java/cloud/pangeacyber/pangea/audit/ITAuditTest.java

```diff
@@ -1169,4 +1171,67 @@ public void testExportDownload() throws PangeaAPIException, PangeaException, Int
 		assertEquals(ResponseStatus.SUCCESS.toString(), downloadResponse.getStatus());
 		assertNotNull(downloadResponse.getResult().getDestURL());
 	}
+
+	@Test
+	public void testMultiThreadStandardEvent() throws PangeaAPIException, PangeaException, InterruptedException {
+		final var taskCount = 200;
+		final var threadsCount = 50;
+
+		final var service = Executors.newFixedThreadPool(threadsCount);
+		final var latch = new CountDownLatch(taskCount);
+		final var totalStartTime = System.nanoTime();
+		for (var i = 0; i < taskCount; i++) {
+			service.submit(() -> {
+				final var event = new StandardEvent.Builder("Hello, World!").action("Login").actor("Terminal").build();
+
+				final var startTime = System.nanoTime();
```

Comment:
nit: You may want to use a monotonic here due to pipeline workers and what not, could be flakey.
`Clock clock = TemporalType.CLOCK.from(SystemClock.MONOTONIC);` from time4j

Reply:
No need to have a exact time here, also I could remove it BTW but I think it could be interesting to have this statistic. I wouldn't want to add a new package just for measure time in this test.